<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>chat</class>
 <widget class="QWidget" name="chat">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_6">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widget_8" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="widget_6" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QWidget" name="widget_4" native="true">
           <layout class="QVBoxLayout" name="verticalLayout_2" stretch="4,0,2">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QWidget" name="widget" native="true">
              <layout class="QVBoxLayout" name="verticalLayout_3">
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item>
                <widget class="QListWidget" name="msgBrowser"/>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QWidget" name="widget_2" native="true">
              <layout class="QVBoxLayout" name="verticalLayout">
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item>
                <widget class="QFrame" name="frame_2">
                 <property name="frameShape">
                  <enum>QFrame::Box</enum>
                 </property>
                 <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,0,0,0,0,0,0,0,0">
                  <property name="leftMargin">
                   <number>0</number>
                  </property>
                  <property name="topMargin">
                   <number>0</number>
                  </property>
                  <property name="rightMargin">
                   <number>0</number>
                  </property>
                  <property name="bottomMargin">
                   <number>0</number>
                  </property>
                  <item>
                   <widget class="QFontComboBox" name="fontCbx"/>
                  </item>
                  <item>
                   <widget class="QComboBox" name="sizeBox">
                    <item>
                     <property name="text">
                      <string>5</string>
                     </property>
                    </item>
                    <item>
                     <property name="text">
                      <string>6</string>
                     </property>
                    </item>
                    <item>
                     <property name="text">
                      <string>7</string>
                     </property>
                    </item>
                    <item>
                     <property name="text">
                      <string>8</string>
                     </property>
                    </item>
                    <item>
                     <property name="text">
                      <string>9</string>
                     </property>
                    </item>
                    <item>
                     <property name="text">
                      <string>10</string>
                     </property>
                    </item>
                    <item>
                     <property name="text">
                      <string>11</string>
                     </property>
                    </item>
                    <item>
                     <property name="text">
                      <string>12</string>
                     </property>
                    </item>
                    <item>
                     <property name="text">
                      <string>20</string>
                     </property>
                    </item>
                    <item>
                     <property name="text">
                      <string>50</string>
                     </property>
                    </item>
                   </widget>
                  </item>
                  <item>
                   <widget class="QToolButton" name="boldBtn">
                    <property name="toolTip">
                     <string>加粗</string>
                    </property>
                    <property name="text">
                     <string>...</string>
                    </property>
                    <property name="icon">
                     <iconset resource="res.qrc">
                      <normaloff>:/image/JC.png</normaloff>:/image/JC.png</iconset>
                    </property>
                    <property name="checkable">
                     <bool>true</bool>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QToolButton" name="italicTbtn">
                    <property name="toolTip">
                     <string>倾斜</string>
                    </property>
                    <property name="text">
                     <string>...</string>
                    </property>
                    <property name="icon">
                     <iconset resource="res.qrc">
                      <normaloff>:/image/qx.png</normaloff>:/image/qx.png</iconset>
                    </property>
                    <property name="checkable">
                     <bool>true</bool>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QToolButton" name="underlineTbtn">
                    <property name="toolTip">
                     <string>下划线</string>
                    </property>
                    <property name="text">
                     <string>...</string>
                    </property>
                    <property name="icon">
                     <iconset resource="res.qrc">
                      <normaloff>:/image/`HJ8@K[456G`KN_9@DOB7WC.png</normaloff>:/image/`HJ8@K[456G`KN_9@DOB7WC.png</iconset>
                    </property>
                    <property name="checkable">
                     <bool>true</bool>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QToolButton" name="colorBtn">
                    <property name="toolTip">
                     <string>颜色</string>
                    </property>
                    <property name="text">
                     <string>...</string>
                    </property>
                    <property name="icon">
                     <iconset resource="res.qrc">
                      <normaloff>:/image/color.png</normaloff>:/image/color.png</iconset>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QToolButton" name="sendfileBtn">
                    <property name="toolTip">
                     <string>发送文件</string>
                    </property>
                    <property name="text">
                     <string>...</string>
                    </property>
                    <property name="icon">
                     <iconset resource="res.qrc">
                      <normaloff>:/image/file.png</normaloff>:/image/file.png</iconset>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QToolButton" name="videobtn">
                    <property name="toolTip">
                     <string>视频聊天</string>
                    </property>
                    <property name="text">
                     <string>...</string>
                    </property>
                    <property name="icon">
                     <iconset resource="res.qrc">
                      <normaloff>:/image/SPTX.png</normaloff>:/image/SPTX.png</iconset>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QToolButton" name="voicebtn">
                    <property name="toolTip">
                     <string>语音聊天</string>
                    </property>
                    <property name="text">
                     <string>...</string>
                    </property>
                    <property name="icon">
                     <iconset resource="res.qrc">
                      <normaloff>:/image/YY.png</normaloff>:/image/YY.png</iconset>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QWidget" name="widget_3" native="true">
              <layout class="QHBoxLayout" name="horizontalLayout_3">
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item>
                <widget class="QTextEdit" name="msgTxtEdit"/>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="widget_5" native="true">
           <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,0">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QPushButton" name="sendBtn">
              <property name="text">
               <string>发送</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>200</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="exitBtn">
              <property name="text">
               <string>退出</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="FIle" native="true">
        <layout class="QVBoxLayout" name="filevlayout">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_7" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
